spec/aggregate_events_spec.rb

```diff
@@ -52,7 +52,7 @@ def old_way_event event_params
       end
 
       it "should store the args to the event" do
-        expect(subject.aggregate_events[1][:event_data][:attribute_deltas][0][:new_value]).to eql("new name")
+        expect(subject.aggregate_events[1][:event_data]["name"][:new_value]).to eql("new name")
```

Comment:
Do we really want to have a string key suddenly? Is it because attributes can have names that aren't valid symbols?

Reply:
I left this as an open question `One question remains and that´s if the keys should be strings or symbols?`